src/main/resources/application.yml

```diff
@@ -0,0 +1,24 @@
+spring:
+  datasource:
+    initialization-mode: always
+    password: ${DATABASE_PASSWORD}
+    platform: postgresql
+    url: jdbc:postgresql://localhost:5432/payment
+    username: ${DATABASE_USERNAME}
+  jpa:
+    database-platform: org.hibernate.dialect.PostgreSQL10Dialect
+    hibernate:
+      ddl-auto: update
```

Comment:
Liquibase already initializes DB, Hibernate shouldn't be in charge of that. Ideally, this should be set to validate

Reply:
was removed